### PR TITLE
docs: remove un-needed use of sheltie

### PR DESCRIPTION
**Pull request by stockholmux**
Fixes #N/A 

**Description**

The troubleshooting guide suggests users of Bottlerocket use `sheltie` to grab kubelet logs. 

Sheltie is just not needed for this as the root file system is always mounted in the admin container on Bottlerocket. 

Sheltie elevates privileges of the user needlessly to accomplish the task described.

**How was this change tested?**

**Does this change impact docs?**
- [x] Yes, PR includes docs updates 
- [ ] Yes, issue opened: # 
- [] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by geoffcline — APPROVED** [review]
LGTM.

**Review by chrisnegus — COMMENTED** [review]
I tried this and it worked as described.
/lgtm

File: website/content/en/docs/troubleshooting.md
Comment (njtran) [inline website/content/en/docs/troubleshooting.md:1]: Can you add this to the preview docs folder as well? We use that for auto-generation for future versions.

Comment (stockholmux) [reply website/content/en/docs/troubleshooting.md:1]: Done!

**Review by jonathan-innis — APPROVED** [review]
LGTM 🚀 Thanks!

**Pull request merged**